source/lib/predicates/boolean.ts

```diff
@@ -13,7 +13,7 @@ export class BooleanPredicate extends Predicate<boolean> {
 	 */
 	get true() {
 		return this.addValidator({
-			message: value => `Expected ${value} to be true`,
+			message: (value, label) => `Expected ${label} ${value} to be true`,
```

Comment:
Should we rephrase this to

> Expected ${label} to be true, got ${value}

Feels better than having

> Expected foo false to be true

Reply:
sounds good :)